<launch>
    <arg name="load_params_from_file" default="true" />        
    <arg name="queue_size" default="5" />
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="upper_body_detector" default="/upper_body_detector/detections" />
    <arg name="odom" default="/odom" />
    <arg name="rgb_image" default="/rgb/image_raw" />    
    <arg name="camera_info_depth" default="/depth_registered/camera_info" />
    <arg name="camera_info_rgb" default="/rgb/camera_info" />
    <arg name="depth_image" default="/depth_registered/image_rect" />

    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>
    
    <node pkg="bounding_boxes" type="bounding_boxes_node" name="bounding_boxes_node" output="log">
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="depth_image" value="$(arg depth_image)" type="string"/>
        <param name="rgb_image" value="$(arg rgb_image)" type="string"/>
        <param name="camera_info_depth" value="$(arg camera_info_depth)" type="string"/>
        <param name="camera_info_rgb" value="$(arg camera_info_rgb)" type="string"/>
        <param name="upper_body_detector" value="$(arg upper_body_detector)" type="string"/>
        <param name="odom" value="$(arg odom)" type="string"/>
    </node>

</launch> 
